### ISSUE TITLE
How do you find the optimal parameters?

### ISSUE THREAD

**Issue by A11en0** [QingruZhang/AdaLoRA#9]
Hi, thanks for your great work. I notice that Adalora adopts a different pre-train model from LoRA, the hyper-parameters must need to research, and how do you find the optimal parameters for the new model?

**Comment by macabdul9** [comment]
Any update on this? @A11en0

**Comment by A11en0** [comment]
Nope, but I guess they choose the initialized parameters from the paper of DeBerta-v3.
